packages/libs/eda/src/lib/core/api/DataClient/index.ts

```diff
@@ -67,11 +67,12 @@ export default class DataClient extends FetchClientWithCredentials {
   // Histogram
   getHistogram(
     computationName: string,
+    pluginName: string, // now handles timeline and histogram
```

Comment:
This probably isn't necessary.

A similar situation exists with lineplot vs. timeseries. We actually never call the timeseries endpoint.

For consistency we should either just call `histogram` here or make the lineplot viz use the appropriate endpoint.

Reply:
I've done the same for lineplot. But noticed we use the generic getVisualization in boxplot. We could use that everywhere tbh!

Reply:
> I've done the same for lineplot. But noticed we use the generic getVisualization in boxplot. We could use that everywhere tbh!

That is something I've been thinking about for a while. It's probably a better way to factor the code.